pint/models/spindown.py

```diff
@@ -25,29 +25,28 @@ def __init__(self):
         # The number of terms in the taylor exapansion of spin freq (F0...FN)
         self.num_spin_terms = maxderivs
 
```

Comment:
I agree.  I think that F* all should be prefix parameters.... (unless F0 has to be treated specially because of precision issues or some other reason).